persistence/relational-jdbc/src/main/java/org/apache/polaris/persistence/relational/jdbc/JdbcMetaStoreManagerFactory.java

```diff
@@ -108,18 +115,19 @@ private void initializeForRealm(
     metaStoreManagerMap.put(realmContext.getRealmIdentifier(), metaStoreManager);
   }
 
-  private DatasourceOperations getDatasourceOperations(boolean isBootstrap) {
+  private DatasourceOperations getDatasourceOperations(Optional<SchemaOptions> schemaOptions) {
     DatasourceOperations databaseOperations;
     try {
       databaseOperations = new DatasourceOperations(dataSource.get(), relationalJdbcConfiguration);
     } catch (SQLException sqlException) {
       throw new RuntimeException(sqlException);
     }
-    if (isBootstrap) {
+    // If this is set, we are bootstrapping
+    if (schemaOptions.isPresent()) {
```

Comment:
Since we're touching this code, I'd like to avoid having bootstrap logic on the main runtime call path (in server). Do you think we could refactor that to execute DDL only in the admin tool?

Reply:
I tried my hand at this in 6fee80f14e4904bc4f43d1849db713d60cd00a2c, let me know if that looks like what you had in mind